modules/rosa-sts-interactive-cluster-creation-mode-options.adoc

```diff
@@ -37,6 +47,12 @@ The following table describes the interactive cluster creation mode options:
 |`Operator roles prefix`
 |Enter a prefix to assign to the cluster-specific Operator IAM roles. The default is the name of the cluster and a 4-digit random string, for example `my-rosa-cluster-a0b1`.
 
+|`Deploy cluster using pre registered OIDC Configuration ID`
+|Specify if you want to use a pre-configured OIDC configuration or if you want to create a new OIDC configuration as part of the cluster creation process.
+
+|`Tags (optional)`
+|Specify a tag that is used on all resources created by ROSA in AWS. Tags are comma separated, for example: "key value, foo bar".
```

Comment:
Should period be inside of end quotation, or outside to prevent someone from putting it in their tag?